src/server/dom.ts

```diff
@@ -367,11 +367,9 @@ export class ElementHandle<T extends Node = Node> extends js.JSHandle<T> {
     return this._retryPointerAction(progress, 'hover', false /* waitForEnabled */, point => this._page.mouse.move(point.x, point.y), options);
   }
 
-  click(options: types.MouseClickOptions & types.PointerActionWaitOptions & types.NavigatingActionWaitOptions = {}): Promise<void> {
-    return this._page._runAbortableTask(async progress => {
```

Comment:
Does it mean that it will now hang on the server side and be aborted only on the client? If so let's remove `timeout` from the options here.

Reply:
Progress (that handles timeout) is now created in the ElementHandleDispatcher instead. Good point about the options, I will follow up with cleaning them up across the board.